src/components/tutor-popover.cjsx

```diff
@@ -0,0 +1,116 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+ArbitraryHtml = require './html'
+
+TutorPopover = React.createClass
```

Comment:
Can this be used as a drop-in replacement for cases when we use `OverlayTrigger` + `BS.Popover`?